content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -730,6 +730,14 @@ Used on: Namespace
 
 This annotation requires the [PodTolerationRestriction](/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction) admission controller to be enabled. This annotation key allows assigning tolerations to a namespace and any new pods created in this namespace would get these tolerations added.
 
+### scheduler.alpha.kubernetes.io/tolerationsWhitelist {#schedulerkubernetestolerations-whitelist}
+
+Example: `scheduler.alpha.kubernetes.io/tolerationsWhitelist: '[{"operator": "Exists", "effect": "NoSchedule", "key": "dedicated-node"}]'`
+
+Used on: Namespace
+
+This annotation requires the [PodTolerationRestriction](/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction) admission controller to be enabled. This annotation value is a JSON document. It's key adds lists of allowed tolerations to the namespace. In case of any conflict between tolerations of a pod and the tolerations of its namespace, the tolerations of the pod is checked against this list. The pod request is admitted only if the check succeeds.
```

Comment:
```suggestion
This annotation is only useful when the (alpha)
[PodTolerationRestriction](/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction)
admission controller is enabled. The annotation value is a JSON document that defines a list of allowed tolerations
for the namespace it annotates. When you create a Pod or modify its tolerations, the API server checks the tolerations
to see if they are mentioned in the allow list. The pod is admitted only if the check succeeds.
```

Reply:
This was almost OK to merge, but:
- all on one line
- The  wording was a bit too far away from idiomatic American English

Try this instead?